sep490-frontend/src/app/modules/enterprise/components/update-user/update-user.component.ts

```
@@ -0,0 +1,192 @@
+      this.enterpriseUserForm.buildings.markAsUntouched();
+    } else {
+      this.enterpriseUserForm.buildings.removeValidators(Validators.required);
+      this.enterpriseUserForm.buildings.markAsUntouched();
+    }
+  }
+  permissionRoleUser(): string {
+    return this.enterpriseUserForm.permissionRole.value?.toString() as string;
+  }
+  protected updateOptionsLabel(): void {
+    this.permissionRoleOptions.forEach(
+      x => (x.name = this.translate.instant(x.i18nCode))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)